R/ggplotly.R

```diff
@@ -814,7 +814,7 @@ gg2list <- function(p, width = NULL, height = NULL,
           bbox(axisTickText, axisObj$tickangle, axisObj$tickfont$size)[[type]] +
           bbox(axisTitleText, axisTitle$angle, unitConvert(axisTitle, "pixels", type))[[type]]
         
-        if (nchar(axisTitleText) > 0) {
+        if (!is.null(axisTitleText) && nchar(axisTitleText) > 0) {
```

Comment:
```suggestion
        if (nzchar(axisTitleText %||% "")) {
```

Reply:
Sure, let me just make sure that behaves as expected with the plot title too.